### PR TITLE
👽️ Include `createdTotalSales` in user stats

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/util/api/likernft/user.ts [hunk 1]
```diff
@@ -43,7 +44,9 @@ export async function getUserStat(wallet) {
     collected_classes: collectedClasses,
     created_count: createdClassCount,
     collector_count: createdCollectorCount,
+    total_sales: totalSalesInNanolike,
   } = userStat;
+  const createdTotalSales = new BigNumber(totalSalesInNanolike).shiftedBy(-9).toNumber();
```
Comment (williamchong) [inline src/util/api/likernft/user.ts]: maybe round to 2dp or integer?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/util/api/likernft/user.ts (hunk 1 above)
Comment (WeiJun0827) [reply src/util/api/likernft/user.ts]: Yeah, can do. I was thinking leaving the decision to front-end to make?

Comment (williamchong) [reply src/util/api/likernft/user.ts]: you can't expect to send a bignumber with `number`, if you want frontend to make decision it needs to be `string`

Comment (WeiJun0827) [reply src/util/api/likernft/user.ts:49]: I see, I'll just return an integer.

Comment (WeiJun0827) [reply src/util/api/likernft/user.ts]: I see, I'll just return an integer.

**Pull request merged**